### PR TITLE
Add new translation for french

### PR DESCRIPTION
**Pull request by detobel36**
Linked to #248

### CODE REVIEW

File: packages/locale/res/translations/fr.json [hunk 1]
```diff
@@ -521,31 +521,31 @@
...
+    "Invitation acceptée"
   ],
   [
     "Wrong confirmation code. Please try again!",
     "Code de confirmation incorrect. Veuillez réessayer !"
   ],
   [
     "Are you sure you want to dismiss this invite?",
-    ""
+    "Vous êtes sûr de vouloir rejeter cette invitation ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:532]:
```suggestion
    "Vous êtes sûr de vouloir rejeter cette invitation?"
```

File: packages/locale/res/translations/fr.json [hunk 2]
```diff
@@ -521,31 +521,31 @@
...
   ],
   [
     "Invite",
-    ""
+    "Invitation"
   ],
   [
     "Invite not found.",
-    ""
+    "Invitation non trouvée"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:548]:
```suggestion
    "Invitation non trouvée."
```

File: packages/locale/res/translations/fr.json [hunk 3]
```diff
@@ -561,43 +561,43 @@
...
   ],
   [
     "{0} invited to join",
-    ""
+    "{0} vous invite à rejoindre"
   ],
   [
     "You have already accepted this invite!",
-    ""
+    "Vous avez déjà accepté cette invitation !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:572]:
```suggestion
    "Vous avez déjà accepté cette invitation!"
```

File: packages/locale/res/translations/fr.json [hunk 4]
```diff
@@ -561,43 +561,43 @@
...
   ],
   [
     "This invite has expired!",
-    ""
+    "Cette invitation est expirée !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:576]:
```suggestion
    "Cette invitation est expirée!"
```

File: packages/locale/res/translations/fr.json [hunk 5]
```diff
@@ -561,43 +561,43 @@
...
   ],
   [
     "Please enter the confirmation code provided to you by the organization owner!",
-    ""
+    "Veuillez entrer le code de confirmation qui vous a été fourni par le propriétaire de l'organisation !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:580]:
```suggestion
    "Veuillez entrer le code de confirmation qui vous a été fourni par le propriétaire de l'organisation!"
```

File: packages/locale/res/translations/fr.json [hunk 6]
```diff
@@ -561,43 +561,43 @@
...
   ],
   [
     "Confirmation Code",
     "Code de confirmation"
   ],
   [
     "Are you sure you want to delete this invite?",
-    ""
+    "Êtes-vous sûr de vouloir supprimer cette invitation ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:588]:
```suggestion
    "Êtes-vous sûr de vouloir supprimer cette invitation?"
```

File: packages/locale/res/translations/fr.json [hunk 7]
```diff
@@ -561,43 +561,43 @@
...
   ],
   [
     "Delete Invite",
-    ""
+    "Supprimer l'invitation"
   ],
   [
     "{0} was successfully added to your organisation!",
-    ""
+    "{0} a bien été ajouté à votre organisation !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:596]:
```suggestion
    "{0} a bien été ajouté à votre organisation!"
```

File: packages/locale/res/translations/fr.json [hunk 8]
```diff
@@ -609,71 +609,71 @@
...
   ],
   [
     "Move To Vault ...",
-    ""
+    "Plus de coffres-forts..."
   ],
   [
     "Delete Item",
-    ""
+    "Supprimer un élément"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:636]:
```suggestion
    "Supprimer l'élément"
```

File: packages/locale/res/translations/fr.json [hunk 9]
```diff
@@ -609,71 +609,71 @@
...
+    "Ajouter un champ"
   ],
   [
     "Attachments",
     "Pièces jointes"
   ],
   [
     "Click or drag files here to add an attachment!",
-    ""
+    "Cliquez ou déplacez les fichiers ici pour ajouter une pièce jointe !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:656]:
```suggestion
    "Cliquez ou déplacez les fichiers ici pour ajouter une pièce jointe!"
```

File: packages/locale/res/translations/fr.json [hunk 10]
```diff
@@ -609,71 +609,71 @@
...
   ],
   [
     "by {0}",
     "par {0}"
   ],
   [
     "Are you sure you want to remove this field?",
-    ""
+    "Êtes-vous sûr de vouloir supprimer ce champ ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:664]:
```suggestion
    "Êtes-vous sûr de vouloir supprimer ce champ?"
```

File: packages/locale/res/translations/fr.json [hunk 11]
```diff
@@ -781,19 +781,19 @@
   ],
   [
     "Do you want to add this device as a trusted device?",
-    ""
+    "Voulez-vous ajouter cet appareil comme périphérique de confiance ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:784]:
```suggestion
    "Voulez-vous ajouter cet appareil comme périphérique de confiance?"
```

File: packages/locale/res/translations/fr.json [hunk 12]
```diff
@@ -781,19 +781,19 @@
...
   ],
   [
     "Add Trusted Device",
-    ""
+    "Ajouter un appareil de confiance"
   ],
   [
     "We failed to verify your email address. Please start over!",
-    ""
+    "Nous n'avons pas réussi à vérifier votre adresse e-mail. Veuillez recommencer !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:792]:
```suggestion
    "Nous n'avons pas réussi à vérifier votre adresse e-mail. Veuillez recommencer!"
```

File: packages/locale/res/translations/fr.json [hunk 13]
```diff
@@ -781,19 +781,19 @@
...
   ],
   [
     "Wrong master password. Please try again!",
-    ""
+    "Mot de passe principal erroné. Veuillez réessayer !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:796]:
```suggestion
    "Mot de passe principal erroné. Veuillez réessayer!"
```

File: packages/locale/res/translations/fr.json [hunk 14]
```diff
@@ -849,7 +849,7 @@
   ],
   [
     "Server error.",
-    ""
+    "Erreur serveur."
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:852]:
```suggestion
    "Erreur de serveur."
```

File: packages/locale/res/translations/fr.json [hunk 15]
```diff
@@ -865,59 +865,59 @@
   ],
   [
     "Account Exists",
-    ""
+    "Le compte existe"
   ],
   [
     "You don't have a Padloc 4 account yet but we've found an account from an older version. Would you like to migrate your account to Padloc 4 now?",
-    ""
+    "Vous n'avez pas encore de compte Padloc 4 mais nous avons trouvé un compte d'une ancienne version. Voulez-vous migrer votre compte vers Padloc 4 maintenant ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:872]:
```suggestion
    "Vous n'avez pas encore de compte Padloc 4 mais nous avons trouvé un compte d'une ancienne version. Voulez-vous migrer votre compte vers Padloc 4 maintenant?"
```

File: packages/locale/res/translations/fr.json [hunk 16]
```diff
@@ -865,59 +865,59 @@
...
   ],
   [
     "Migrating Account",
-    ""
+    "Migration du compte"
   ],
   [
     "Your account and all associated data was migrated successfully! Do you want to delete your old account now?",
-    ""
+    "Votre compte et toutes les données associées ont été migrés avec succès ! Voulez-vous supprimer votre ancien compte maintenant ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:888]:
```suggestion
    "Votre compte et toutes les données associées ont été migrés avec succès ! Voulez-vous supprimer votre ancien compte maintenant?"
```

File: packages/locale/res/translations/fr.json [hunk 17]
```diff
@@ -865,59 +865,59 @@
...
   ],
   [
     "Delete Legacy Account",
-    ""
+    "Supprimer l'ancien compte"
   ],
   [
     "All done! Please note that you won't be able to access your Padloc 4 account with older versions of the app, so please make sure you have the latest version installed on all your devices! (You can find download links for all platforms at https://padloc.app/downloads/). Enjoy using Padloc 4!",
-    ""
+    "C'est fait ! Veuillez noter que vous ne pourrez pas accéder à votre compte Padloc 4 avec les anciennes versions de l'application, donc assurez-vous d'avoir la dernière version installée sur tous vos appareils ! (Vous pouvez trouver les liens de téléchargement pour toutes les plateformes sur https://padloc.app/downloads/). Profitez de Padloc 4 !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:896]:
```suggestion
    "C'est fait ! Veuillez noter que vous ne pourrez pas accéder à votre compte Padloc 4 avec les anciennes versions de l'application, donc assurez-vous d'avoir la dernière version installée sur tous vos appareils! (Vous pouvez trouver les liens de téléchargement pour toutes les plateformes sur https://padloc.app/downloads/). Profitez de Padloc 4!"
```

File: packages/locale/res/translations/fr.json [hunk 18]
```diff
@@ -937,11 +937,11 @@
   ],
   [
     "This random passphrase was generated just for you and is designed to be both secure and easy to remember.",
-    ""
+    "Cette phrase de mot de passe aléatoire a été générée juste pour vous et est conçue pour être à la fois sûre et facile à retenir."
   ],
   [
     "Don't like it?",
-    ""
+    "Vous ne l'aimez pas ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:944]:
```suggestion
    "Vous ne l'aimez pas?"
```

File: packages/locale/res/translations/fr.json [hunk 19]
```diff
@@ -953,31 +953,31 @@
...
+    "Changer de nom"
   ],
   [
     "Repeat Master Password",
     "Répéter le mot de passe principal"
   ],
   [
     "Your master password is the last password you'll ever have to remember! Please memorize it and never reveal it to anyone - not even us! We recommend writing it down on a piece of paper and storing it somewhere safe, at least until you have it safely memorized.",
-    ""
+    "Votre mot de passe principal est le dernier mot de passe que vous aurez à retenir ! Mémorisez-le et ne le révélez jamais à personne, pas même à nous ! Nous vous recommandons de l'écrire sur un morceau de papier et de le conserver dans un endroit sûr, au moins jusqu'à ce que vous l'ayez mémorisé."
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:964]:
```suggestion
    "Votre mot de passe principal est le dernier mot de passe que vous aurez à retenir! Mémorisez-le et ne le révélez jamais à personne, pas même à nous! Nous vous recommandons de l'écrire sur un morceau de papier et de le conserver dans un endroit sûr, au moins jusqu'à ce que vous l'ayez mémorisé."
```

File: packages/locale/res/translations/fr.json [hunk 20]
```diff
@@ -953,31 +953,31 @@
...
   ],
   [
     "Change Password",
-    ""
+    "Changer le mot de passe"
   ],
   [
     "All set!",
-    ""
+    "Tout est prêt !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:972]:
```suggestion
    "Tout est prêt!"
```

File: packages/locale/res/translations/fr.json [hunk 21]
```diff
@@ -953,31 +953,31 @@
...
   ],
   [
     "Your account was created successfully. Enjoy using {0}!",
-    ""
+    "Votre compte a été créé avec succès. Profitez de l'utilisation de {0} !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:976]:
```suggestion
    "Votre compte a été créé avec succès. Profitez de l'utilisation de {0}!"
```

File: packages/locale/res/translations/fr.json [hunk 22]
```diff
@@ -989,15 +989,15 @@
...
+    "Provisionné"
   ],
   [
     "Suspended",
     "Suspendu"
   ],
   [
     "Something went wrong while processing your request. Please try again later!",
-    ""
+    "Un problème est survenu lors du traitement de votre demande. Veuillez réessayer plus tard !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1000]:
```suggestion
    "Un problème est survenu lors du traitement de votre demande. Veuillez réessayer plus tard!"
```

File: packages/locale/res/translations/fr.json [hunk 23]
```diff
@@ -1013,15 +1013,15 @@
   ],
   [
     "Are you sure you want to transfer this organizations ownership to {0}?",
-    ""
+    "Etes-vous sûr de vouloir transférer la propriété de cette organisation à {0} ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1016]:
```suggestion
    "Etes-vous sûr de vouloir transférer la propriété de cette organisation à {0}?"
```

File: packages/locale/res/translations/fr.json [hunk 24]
```diff
@@ -1013,15 +1013,15 @@
...
   ],
   [
     "Make Owner",
-    ""
+    "Définir comme propriétaire"
   ],
   [
     "The organization ownership was transferred successfully!",
-    ""
+    "La propriété de l'organisation a été transférée avec succès !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1024]:
```suggestion
    "La propriété de l'organisation a été transférée avec succès!"
```

File: packages/locale/res/translations/fr.json [hunk 25]
```diff
@@ -1057,63 +1057,63 @@
...
   ],
   [
     "Invites",
-    ""
+    "Invitations"
   ],
   [
     "A newer version of {0} is required to synchronize this vault. Please update to the latest version now!",
-    ""
+    "Une version plus récente de {0} est nécessaire pour synchroniser ce coffre-fort. Veuillez effectuer la mise à jour vers la dernière version maintenant !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1100]:
```suggestion
    "Une version plus récente de {0} est nécessaire pour synchroniser ce coffre-fort. Veuillez effectuer la mise à jour vers la dernière version maintenant!"
```

File: packages/locale/res/translations/fr.json [hunk 26]
```diff
@@ -1265,11 +1265,11 @@
   ],
   [
     "Do you want to enable Directory Sync via SCIM for this organization? You will be given a unique URL to provide to your Active Directory or LDAP server for synchronizing and provisioning members.",
-    ""
+    "Voulez-vous activer la synchronisation des répertoires via SCIM pour cette organisation ? Vous recevrez une URL unique à fournir à votre Active Directory ou à votre serveur LDAP pour la synchronisation et le provisionnement des membres."
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1268]:
```suggestion
    "Voulez-vous activer la synchronisation des répertoires via SCIM pour cette organisation? Vous recevrez une URL unique à fournir à votre Active Directory ou à votre serveur LDAP pour la synchronisation et le provisionnement des membres."
```

File: packages/locale/res/translations/fr.json [hunk 27]
```diff
@@ -1265,11 +1265,11 @@
...
   ],
   [
     "Do you want to disable Directory Sync? Your members will no longer be automatically synchronized and provisioned.",
-    ""
+    "Voulez-vous désactiver Directory Sync ? Vos membres ne seront plus synchronisés et provisionnés automatiquement."
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1272]:
```suggestion
    "Voulez-vous désactiver Directory Sync? Vos membres ne seront plus synchronisés et provisionnés automatiquement."
```

File: packages/locale/res/translations/fr.json [hunk 28]
```diff
@@ -1337,7 +1337,7 @@
   ],
   [
     "IMPORTANT, READ CAREFULLY: {0} is designed in a way that makes it impossible for us to access the data encrypted in any of your vaults even if we wanted to. While this is essential to ensuring the security of your data, it also has the following implications:",
-    ""
+    "IMPORTANT, LIRE ATTENTIVEMENT: {0} est conçu de manière à ce qu'il nous soit impossible d'accéder aux données chiffré dans l'un de vos coffres-forts, même si nous le voulions. Bien que cela soit essentiel pour assurer la sécurité de vos données, cela a également les implications suivantes :"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1340]:
```suggestion
    "IMPORTANT, LIRE ATTENTIVEMENT: {0} est conçu de manière à ce qu'il nous soit impossible d'accéder aux données chiffré dans l'un de vos coffres-forts, même si nous le voulions. Bien que cela soit essentiel pour assurer la sécurité de vos données, cela a également les implications suivantes:"
```

File: packages/locale/res/translations/fr.json [hunk 29]
```diff
@@ -1445,75 +1445,75 @@
   ],
   [
     "The following organizations are owned by you and will be deleted along with your account:",
-    ""
+    "Les organisations suivantes vous appartiennent et seront supprimées en même temps que votre compte :"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1448]:
```suggestion
    "Les organisations suivantes vous appartiennent et seront supprimées en même temps que votre compte:"
```

File: packages/locale/res/translations/fr.json [hunk 30]
```diff
@@ -1553,159 +1553,159 @@
   ],
   [
     "Are you sure you want to disable biometric unlock for this device?",
-    ""
+    "Êtes-vous sûr de vouloir désactiver le déverrouillage biométrique pour cet appareil ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1556]:
```suggestion
    "Êtes-vous sûr de vouloir désactiver le déverrouillage biométrique pour cet appareil?"
```

File: packages/locale/res/translations/fr.json [hunk 31]
```diff
@@ -1553,159 +1553,159 @@
...
   ],
   [
     "Disable Biometric Unlock",
-    ""
+    "Désactiver le déverrouillage biométrique"
   ],
   [
     "What kind of multi-factor authenticator would you like to add?",
-    ""
+    "Quel type d'authentification multi-facteurs souhaitez-vous ajouter ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1568]:
```suggestion
    "Quel type d'authentification multi-facteurs souhaitez-vous ajouter?"
```

File: packages/locale/res/translations/fr.json [hunk 32]
```diff
@@ -1553,159 +1553,159 @@
...
   ],
   [
     "Failed to add authenticator",
-    ""
+    "Échec de l'ajout d'un authentificateur"
   ],
   [
     "Are you sure you want to delete this authenticator?",
-    ""
+    "Êtes-vous sûr de vouloir supprimer cet authentificateur ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1576]:
```suggestion
    "Êtes-vous sûr de vouloir supprimer cet authentificateur?"
```

File: packages/locale/res/translations/fr.json [hunk 33]
```diff
@@ -1553,159 +1553,159 @@
...
   ],
   [
     "Delete Authenticator",
-    ""
+    "Supprimer l'authentificateur"
   ],
   [
     "Are you sure you want to revoke this session?",
-    ""
+    "Êtes-vous sûr de vouloir révoquer cette session ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1584]:
```suggestion
    "Êtes-vous sûr de vouloir révoquer cette session?"
```

File: packages/locale/res/translations/fr.json [hunk 34]
```diff
@@ -1553,159 +1553,159 @@
...
   ],
   [
     "Revoke Session",
-    ""
+    "Révoquer la session"
   ],
   [
     "Are you sure you want to remove this device from your trusted devices?",
-    ""
+    "Êtes-vous sûr de vouloir supprimer cet appareil de vos appareils de confiance ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1596]:
```suggestion
    "Êtes-vous sûr de vouloir supprimer cet appareil de vos appareils de confiance?"
```

File: packages/locale/res/translations/fr.json [hunk 35]
```diff
@@ -1553,159 +1553,159 @@
...
   ],
   [
     "Remove Trusted Device",
-    ""
+    "Supprimer l'appareil de confiance"
   ],
   [
     "The test was successfull!",
-    ""
+    "Le test est réussi !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1604]:
```suggestion
    "Le test est réussi!"
```

File: packages/locale/res/translations/fr.json [hunk 36]
```diff
@@ -1553,159 +1553,159 @@
...
   ],
   [
     "Test Authenticator",
-    ""
+    "Test de l'authentificateur"
   ],
   [
     "Test failed!",
-    ""
+    "Le test a échoué !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1612]:
```suggestion
    "Le test a échoué!"
```

File: packages/locale/res/translations/fr.json [hunk 37]
```diff
@@ -1553,159 +1553,159 @@
...
   ],
   [
     "Test failed! Error: {0}",
-    ""
+    "Le test a échoué ! Erreur: {0}"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1616]:
```suggestion
    "Le test a échoué! Erreur: {0}"
```

File: packages/locale/res/translations/fr.json [hunk 38]
```diff
@@ -1553,159 +1553,159 @@
...
   ],
   [
     "Are you sure you want to revoke biometric unlock for the device \"{0}\"?",
-    ""
+    "Êtes-vous sûr de vouloir révoquer le déverrouillage biométrique de l'appareil \"{0}\" ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1620]:
```suggestion
    "Êtes-vous sûr de vouloir révoquer le déverrouillage biométrique de l'appareil \"{0}\"?"
```

File: packages/locale/res/translations/fr.json [hunk 39]
```diff
@@ -1765,7 +1765,7 @@
   ],
   [
     "Welcome back! Please enter your master password to unlock the app.",
-    ""
+    "Bon retour ! Veuillez entrer votre mot de passe principal pour déverrouiller l'application."
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1768]:
```suggestion
    "Bon retour! Veuillez entrer votre mot de passe principal pour déverrouiller l'application."
```

File: packages/locale/res/translations/fr.json [hunk 40]
```diff
@@ -1829,23 +1829,23 @@
   ],
   [
     "Please enter a Vault name!",
-    ""
+    "Veuillez entrer un nom du coffre-fort !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1832]:
```suggestion
    "Veuillez entrer un nom du coffre-fort!"
```

File: packages/locale/res/translations/fr.json [hunk 41]
```diff
@@ -1829,23 +1829,23 @@
...
   ],
   [
     "Vault name required!",
-    ""
+    "Nom du coffre-fort requis !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1836]:
```suggestion
    "Nom du coffre-fort requis!"
```

File: packages/locale/res/translations/fr.json [hunk 42]
```diff
@@ -1829,23 +1829,23 @@
...
   ],
   [
     "Please enter a different Vault name!",
-    ""
+    "Veuillez entrer un autre nom de coffre-fort !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1840]:
```suggestion
    "Veuillez entrer un autre nom de coffre-fort!"
```

File: packages/locale/res/translations/fr.json [hunk 43]
```diff
@@ -1829,23 +1829,23 @@
...
   ],
   [
     "Reserved Name!",
-    ""
+    "Nom réservé !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1844]:
```suggestion
    "Nom réservé!"
```

File: packages/locale/res/translations/fr.json [hunk 44]
```diff
@@ -1829,23 +1829,23 @@
...
   ],
   [
     "Please assign at least on member or group to this vault!",
-    ""
+    "Veuillez affecter au moins un membre ou un groupe à ce coffre-fort !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1848]:
```suggestion
    "Veuillez affecter au moins un membre ou un groupe à ce coffre-fort!"
```

File: packages/locale/res/translations/fr.json [hunk 45]
```diff
@@ -1857,59 +1857,59 @@
   ],
   [
     "Vault deleted successfully!",
-    ""
+    "Coffre-fort supprimée avec succès !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1860]:
```suggestion
    "Coffre-fort supprimée avec succès!"
```

File: packages/locale/res/translations/fr.json [hunk 46]
```diff
@@ -1857,59 +1857,59 @@
...
   ],
   [
     "No Members have been given access to this vault yet.",
-    ""
+    "Aucun membre n'a encore eu accès à ce coffre-fort."
   ],
   [
     "The app could not establish a connection with our servers, please check your internet connection or try again later!",
-    ""
+    "L'application n'a pas pu établir de connexion avec nos serveurs, veuillez vérifier votre connexion Internet ou réessayer plus tard !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1876]:
```suggestion
    "L'application n'a pas pu établir de connexion avec nos serveurs, veuillez vérifier votre connexion Internet ou réessayer plus tard!"
```

File: packages/locale/res/translations/fr.json [hunk 47]
```diff
@@ -1857,59 +1857,59 @@
...
   ],
   [
     "The app could not establish a connection with the our servers, please check your internet connection or try again later!",
-    ""
+    "L'application n'a pas pu établir de connexion avec nos serveurs, veuillez vérifier votre connexion Internet ou réessayer plus tard !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1880]:
```suggestion
    "L'application n'a pas pu établir de connexion avec nos serveurs, veuillez vérifier votre connexion Internet ou réessayer plus tard!"
```

File: packages/locale/res/translations/fr.json [hunk 48]
```diff
@@ -1857,59 +1857,59 @@
...
   ],
   [
     "You don't have any items with weak passwords!",
-    ""
+    "Vous n'avez pas d'éléments avec des mots de passe faibles !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1884]:
```suggestion
    "Vous n'avez pas d'éléments avec des mots de passe faibles!"
```

File: packages/locale/res/translations/fr.json [hunk 49]
```diff
@@ -1857,59 +1857,59 @@
...
   ],
   [
     "You don't have any items with reused passwords!",
-    ""
+    "Vous n'avez pas d'éléments avec des mots de passe réutilisés !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1888]:
```suggestion
    "Vous n'avez pas d'éléments avec des mots de passe réutilisés!"
```

File: packages/locale/res/translations/fr.json [hunk 50]
```diff
@@ -1857,59 +1857,59 @@
...
   ],
   [
     "You don't have any items with compromised passwords!",
-    ""
+    "Vous n'avez pas d'éléments avec des mots de passe compromis !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1892]:
```suggestion
    "Vous n'avez pas d'éléments avec des mots de passe compromis!"
```

File: packages/locale/res/translations/fr.json [hunk 51]
```diff
@@ -1857,59 +1857,59 @@
...
   ],
   [
     "You don't have any insecure items!",
-    ""
+    "Vous n'avez pas d'éléments non sécurisés !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1896]:
```suggestion
    "Vous n'avez pas d'éléments non sécurisés!"
```

File: packages/locale/res/translations/fr.json [hunk 52]
```diff
@@ -1925,67 +1925,67 @@
   ],
   [
     "Authentication type not supported!",
-    ""
+    "Le type d'authentification n'est pas pris en charge !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1928]:
```suggestion
    "Le type d'authentification n'est pas pris en charge!"
```

File: packages/locale/res/translations/fr.json [hunk 53]
```diff
@@ -1925,67 +1925,67 @@
...
+    "Accès refusé"
   ],
   [
     "Your session has expired. Please log in again!",
     "Votre session est terminée. Veuillez vous reconnecter !"
   ],
   [
     "Some data associated with your account was saved with a newer version of Padloc and cannot be decoded. Please install the latest version Padloc!",
-    ""
+    "Certaines données associées à votre compte ont été enregistrées avec une version plus récente de Padloc et ne peuvent être décodées. Veuillez installer la dernière version de Padloc !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1976]:
```suggestion
    "Certaines données associées à votre compte ont été enregistrées avec une version plus récente de Padloc et ne peuvent être décodées. Veuillez installer la dernière version de Padloc!"
```

File: packages/locale/res/translations/fr.json [hunk 54]
```diff
@@ -1925,67 +1925,67 @@
...
+    "Compte suspendu"
   ],
   [
     "Access Denied",
-    ""
+    "Accès refusé"
   ],
   [
     "Your session has expired. Please log in again!",
     "Votre session est terminée. Veuillez vous reconnecter !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1972]:
```suggestion
    "Votre session est terminée. Veuillez vous reconnecter!"
```

File: packages/locale/res/translations/fr.json [hunk 55]
```diff
@@ -1925,67 +1925,67 @@
...
   ],
   [
     "Are you sure you want to leave this page? Any changes will be lost.",
-    ""
+    "Êtes-vous sûr de vouloir quitter cette page ? Toute modification sera perdue."
   ],
   [
     "A new update is ready to install! Do you want to install it now?",
-    "Une nouvelle mise à jour est prête à être installée ! Voulez vous l'installer maintenant ?"
+    "Une nouvelle mise à jour est prête à être installée ! Voulez-vous l'installer maintenant ?"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1988]:
```suggestion
    "Une nouvelle mise à jour est prête à être installée! Voulez-vous l'installer maintenant?"
```

File: packages/locale/res/translations/fr.json [hunk 56]
```diff
@@ -1925,67 +1925,67 @@
...
   ],
   [
     "Update Required",
-    ""
+    "Mise à jour requise"
   ],
   [
     "Are you sure you want to leave this page? Any changes will be lost.",
-    ""
+    "Êtes-vous sûr de vouloir quitter cette page ? Toute modification sera perdue."
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:1984]:
```suggestion
    "Êtes-vous sûr de vouloir quitter cette page? Toute modification sera perdue."
```

File: packages/locale/res/translations/fr.json [hunk 57]
```diff
@@ -1997,31 +1997,31 @@
...
   ],
   [
     "Synching local changes failed because you don't have write permissions for this vault.",
-    ""
+    "La synchronisation des modifications locales a échoué car vous n'avez pas le droit d'écrire dans ce coffre-fort."
   ],
   [
     "This group name is not available!",
-    ""
+    "Ce nom de groupe n'est pas disponible !"
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/fr.json:2020]:
```suggestion
    "Ce nom de groupe n'est pas disponible!"
```

**Review by BrunoBernardino — CHANGES_REQUESTED** [review]
Thank you so much for these @detobel36 ! I've made some suggestions to keep the formatting consistent (no spaces before marks, and ensure both the original and the translations match in terms of sentence-ending mark). I also caught what looked like a couple of typos, which you're welcome to not accept, if they're already correct.

**Comment by detobel36** [comment]
@BrunoBernardino but in french they are space before marks 😕   So if you want to write "good french" you need to keep space.

For other remarks, I'm agree with you.  I will do the change this evening

File: packages/locale/res/translations/fr.json (hunk 14 above)
Comment (detobel36) [reply packages/locale/res/translations/fr.json:852]: Then it's better `Erreur du serveur.`

**Comment by detobel36** [comment]
I have made a fixup.
Before to "merge" I can "squash" commit (or you can do it).

**Comment by BrunoBernardino** [comment]
Merci beaucoup @detobel36 !

**Comment by MaKleSoft** [comment]
@detobel36 Thank you so much for your contribution! Are you twitter by any chance? Would like to give you a shout-out 😊

**Comment by detobel36** [comment]
Yes just "@Detobel36" on twitter 😉 

But thanks to you for this project !

**Pull request merged**